Tasks/AzureRmWebAppDeploymentV4/Tests/AzureRmWebAppDeploymentProviderTests.ts

```diff
@@ -0,0 +1,46 @@
+import tl = require('vsts-task-lib');
+import tmrm = require('vsts-task-lib/mock-run');
+import ma = require('vsts-task-lib/mock-answer');
+import * as path from 'path';
+import { AzureResourceFilterUtility } from '../operations/AzureResourceFilterUtility';
+import { KuduServiceUtility } from '../operations/KuduServiceUtility';
+import { AzureEndpoint } from 'azure-arm-rest/azureModels';
+import { ApplicationTokenCredentials } from 'azure-arm-rest/azure-arm-common';
+import { AzureRMEndpoint } from 'azure-arm-rest/azure-arm-endpoint'; 
+import { setEndpointData, setAgentsData, mockTaskArgument } from './utils';
+
+export class AzureRmWebAppDeploymentProviderTests {
+
+    public static startAzureRmWebAppDeploymentProviderTests(){
+        let tp = path.join(__dirname, 'AzureRmWebAppDeploymentProviderL0Tests.js');
+        let tr : tmrm.TaskMockRunner = new tmrm.TaskMockRunner(tp);
```

Comment:
Can we have a common tr object with default values and override wherever necessary. I see similar object in all files, we can remove some redundancy.

Reply:
Moved into common utils through this commit https://github.com/Microsoft/vsts-tasks/pull/8027/commits/bb951457d8f8ae91b3b62faec2bde2cdd4f9787f